src/control/cmd/daos/filesystem.go

```diff
@@ -292,6 +294,7 @@ func (cmd *fsGetAttrCmd) Execute(_ []string) error {
 		if C.mode_is_dir(cmode) {
 			jsonAttrs := struct {
 				ObjAttr struct {
+					OID      string `json:"OID"`
```

Comment:
nit: Our convention is that JSON keys should be `snake_case`, so this should just be `oid`.